problemsets-mat223/modules/module12-exercises.tex

```diff
@@ -1,17 +1,41 @@
 \begin{exercises}
+    %Patrick:
+        % 60min
 	\begin{problist}
-		\prob  Example problem with parts
+		\prob  Indicate whether the following linear transformations are surjective and/or injective, as well as indicate whether they are invertible. Justify your answers.
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
+			\item   $\mathcal S:\R^2\to\R^2$, a linear transformation that doubles every vector.
```

Comment:
Rephrase all of these to be like `$\mathcal S:\R^2\to\R^2$, where $S$ is the linear transformation that...`